### PR TITLE
integrations: Add screenshots and notes for papertrail

### PR DESCRIPTION
**Pull request by anirudhjain75**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@anirudhjain75, it looks like you've already signed the Dropbox CLA. Thanks!

File: webhook-integrations/papertrail/notes.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    Papertrail log extract works for nearly every device or software.
+    Logs are sent via link that is generated by the integrated tools.
+    I've used a tool named Slack, created an account (team),
+    Searched for papertrail and generated a link posting the logs
+    in #general(group), Made an account in papertrailapp.com
+    and configured it using my pc's logs and set an alert for 
+    the link that was generated by slack, And therefore it worked.
+    This is the way papertrail works.
+    
+Creating a slack account in fig Task_4_5.
```
Comment (pweaver) [inline webhook-integrations/papertrail/notes.md]: Can you convert the Task_<foo> to markdown. E.g. Create Account?

**Comment by pweaver** [comment]
The content looks good!

Can you add the images to the Markdown? You can preview the Notes.md on Github to see what is looks like.

File: webhook-integrations/papertrail/notes.md (hunk 1 above)
Comment (anirudhjain75) [inline webhook-integrations/papertrail/notes.md]: Nope

**Comment by anirudhjain75** [comment]
@pweaver I don't know how to do it.

**Comment by pweaver** [comment]
Sorry, I updated the comment so you can see the markdown.

**Comment by pweaver** [comment]
See https://guides.github.com/features/mastering-markdown/ for how to format in markdown.

**Pull request closed without merging**